test/framework/test_composite.py

```diff
@@ -365,6 +367,35 @@ def test_composite_tags(self):
         self.assertEqual(sorted(data1.tags), ["c", "d"])
         self.assertEqual(sorted(data2.tags), ["c", "d"])
 
+    def test_composite_figures(self):
+        """
+        Test adding figures from composite experiments
+        """
+        exp1 = FakeExperiment([0, 2])
+        exp2 = FakeExperiment([1, 3])
+        exp1.analysis.set_options(add_figures=True)
+        exp2.analysis.set_options(add_figures=True)
+        par_exp = BatchExperiment([exp1, exp2], flatten_results=True)
+        expdata = par_exp.run(FakeBackend())
+        expdata.service = IBMExperimentService(local=True, local_save=False)
+        expdata.auto_save = True
+        expdata.block_for_results()
+        self.assertExperimentDone(expdata)
+
+    def test_composite_auto_save(self):
+        """
+        Test setting autosave when using composite experiments
+        """
+        service = mock.create_autospec(IBMExperimentService, instance=True)
+        exp1 = FakeExperiment([0, 2])
+        exp2 = FakeExperiment([1, 3])
+        par_exp = BatchExperiment([exp1, exp2], flatten_results=False)
+        expdata = par_exp.run(FakeBackend())
+        expdata.service = service
+        expdata.block_for_results()
```

Comment:
Better to use `assertExperimentDone` instead.

Reply:
Done